src/network_analysis.jl

```diff
@@ -628,6 +692,7 @@ conservation laws, each represented as a row in the output.
 function conservationlaws(nsm::T; col_order = nothing) where {T <: AbstractMatrix}
 
     # compute the left nullspace over the integers
+    # The `nullspace` function updates the `col_order`.
```

Comment:
Is this true @isaacsas? I tried tracking this variable throughout, and looked into the `nullspace` function, int this is my impression of what happens.
(I was unsure since they call it `nullspace` and not `nullspace!`)